### ISSUE TITLE
Playlist mode

### ISSUE THREAD

**Issue by matejdro** [dr1rrb/py-twinkly-client#1]
Hello,

currently this client seems to be hardcoded to start lights in the `movie` mode: https://github.com/dr1rrb/py-twinkly-client/blob/main/twinkly_client/client.py#L52

I would like to add an option to HA component to start the lights in `playlist` mode instead (mode where you set up a playlist in the Twinkly app and then lights follow that sequence). 

What would be the best way to proceed with this? Should I add parameter to the ` set_is_on` method and then add a `twinkly.turn_on_in_movie` service to HA?

**Comment by Mr-Groch** [comment]
Hi, I'm resurrecting this issue :) Any chance to fix this in near feature @dr1rrb ?

I've found @matejdro fork with workaround that is working:
https://github.com/matejdro/py-twinkly-client/tree/patch-1

Thx @matejdro !

But it would be great to fix this in HA main branch. Manuall change in core .py files each HA update is frustrating...

**Comment by matejdro** [comment]
This patch is not complete though as it always forces lights into playlist mode, even if you have no playlist set (in which case I guess turning on lights won't work at all). A better solution would need to be achieved (maybe a configuration option on what to do when turning on?)

**Comment by dr1rrb** [comment]
Yes I didn't validated that by the time I was developing that client, it's definitely a design error 😓. But I don't think it would make sense to fix it in my poor twinky-client, we should move to a more powerful client like proposed in that PR https://github.com/home-assistant/core/pull/62337
